core/Cargo.toml

```
@@ -57,6 +57,9 @@ chrono = { version = "0.4", features = ["serde"] }
 color-eyre = "*"
 lipsum = "0.8.2"
 
+tower = { version = "0.4", features = ["default"] }
```

```suggestion
tower = "0.4"
```

If we're using default then we can just state the version directly

---

Done :+1: